src/Processors/QueryPlan/Streaming/AggregatingStep.h

```diff
@@ -22,7 +24,8 @@ class AggregatingStep : public ITransformingStep
         size_t merge_threads_,
         size_t temporary_data_merge_threads_,
         bool emit_version_,
-        bool emit_changelog_);
+        bool emit_changelog_,
```

Comment:
`bool emit_changelog_` is redundent moving forward since emit strategy shall cover that

Reply:
do it in a separate PR